electron-app/postbuild_tests/modules.test.ts

```diff
@@ -547,17 +555,73 @@ describe("modules", () => {
     [
       // NOTE: This test relies on the remote module jsbrittain/snakeshack (Utilty) touch
       "(single_modules) container_touch",
-      path.join("utility_touch", "data.csv"),
+      [
+        // target files
+        path.join("results", "utility_touch", "data.csv"),
+      ],
+      [
+        // packaged payload files
+      ],
     ],
   ])(
     "Build, extract zip, run in Docker: module '%s'",
-    async (modulename, outfile) => {
+    async (modulename, target_files, payload_files) => {
       await Build_RunWithDocker_SingleModuleWorkflow(
         driver,
         modulename,
-        outfile,
+        target_files,
+        payload_files,
+        true, // expand_module
       );
     },
     20 * ONE_MINUTE,
   ); // long timeout
+
+  // Package workflow (container test)
+  runif(is_installed(["docker"])).each([
+    [
+      "(single_modules) payload run",
+      [
+        // target files
+        path.join("results", "single_modules_payload_run", "data.csv"),
+      ],
+      [
+        // packaged payload files
+        path.join(
+          "workflow",
+          "modules",
+          "test-repo",
+          "workflows",
+          "single_modules",
+          "sources",
+          "payload_run",
+          "resources",
+          "file",
+        ),
+      ],
+    ],
+  ])(
+    "Package workflow (container): module '%s'",
+    async (modulename, target_files, payload_files) => {
+      // Set workflow packaging option
+      await driver.findElement(By.id("btnSidenavSettings")).click();
+      await SetCheckBoxByID(driver, "package_modules_in_workflow", true);
+      await driver.findElement(By.id("btnSidenavBuilder")).click();
+
+      // Build and run workflow
+      await Build_RunWithDocker_SingleModuleWorkflow(
+        driver,
+        modulename,
+        target_files,
+        payload_files,
+        false, // expand_module
```

Comment:
does typescript have named arguments?

Reply:
Not directly, but you can use dictionaries / destructuring as a workaround (see https://stackoverflow.com/a/42108988). I will rework this for clarity.